### PR TITLE
Install setroubleshoot helpers

### LINKED ISSUE #327: Manage setroubleshootd/sealert?
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 6
- Ruby: 2
- Distribution: EL7
- Module version: 3.2.0

## Any additional information you'd like to impart

Could this module be extended to also manage setroubleshootd (sometimes packaged as sealert) as well?

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description

A sample set of methods to install setroubleshoot (and plugins).

Odds are this PR needs work, it is mostly a 'step one' towards seeing if this is a good fit here.

#### This Pull Request (PR) fixes the following issues
Fixes #327

### CODE REVIEW

File: manifests/package.pp [hunk 1]
```diff
@@ -20,4 +22,7 @@
   if $manage_auditd_package {
     ensure_packages ($auditd_package_name)
   }
+  if $manage_setroubleshoot_packages, {
+    ensure_packages ($setroubleshoot_package_names,)
```
Comment (kenyon) [inline manifests/package.pp:26]: extra comma?

**Review by kenyon — COMMENTED** [review]
Seems like a good idea to me. Just need tests.

File: manifests/package.pp (hunk 1 above)
Comment (jcpunk) [reply manifests/package.pp]: I'll push with a fix.

**Comment by jcpunk** [comment]
Pushed up new version with unit tests.

File: manifests/package.pp [hunk 2]
```diff
@@ -20,4 +22,7 @@
   if $manage_auditd_package {
     ensure_packages ($auditd_package_name)
   }
+  if $manage_setroubleshoot_packages, {
```
Comment (kenyon) [inline manifests/package.pp:25]: extra comma

File: spec/classes/selinux_package_spec.rb [hunk 3]
```diff
@@ -115,4 +115,24 @@
...
+      let(:params) do
+        {
+          manage_setroubleshoot_packages: true
+        }
+      end
+
+      it { is_expected.to contain_package('setroubleshoot').with(ensure: 'present') }
+      it { is_expected.to contain_package('setroubleshoot-plugins').with(ensure: 'present') }
+    end
+
```
Comment (kenyon) [inline spec/classes/selinux_package_spec.rb:137]: extra line here is causing the rubocop test to fail

**Review by kenyon — COMMENTED** [review]
Looks like more work is needed to get the tests to pass.

**Comment by jcpunk** [comment]
I'm a bit confused why the tests are failing....

Shouldn't the spec tests be getting the defaults from Hiera if they are not set as `params`?

**Review by kenyon — APPROVED** [review]
@jcpunk have a look at my changes. I think this is what should be done. (I made them as separate commits so it's easy to see what I did, but we'll want to squash them on merge.)

**Comment by jcpunk** [comment]
ahh, that makes sense when I look at it.

**Pull request merged**